diff -Nur admin1/info.sh admin2/info.sh
--- admin1/info.sh      2012-08-18 01:36:08.824952168 +0400
+++ admin2/info.sh      2012-08-18 01:15:54.532924460 +0400
@@ -19,6 +19,13 @@
     unzip -l "$1"
 }

+handle_pem()
+{
+FP=`openssl x509 -noout -text -fingerprint -in "$1" | grep -E "SHA1 Fingerprint=" | sed 's/SHA1 Fingerprint=//'`
+SUB=`openssl x509 -noout -text -in "$1" | grep -E "Subject:"| sed 's/Subject://' | sed 's/^[ \t]*//'`
+echo "$FP   $SUB"
+}
+

 #
 # Main
